src/hb-subset-input.cc

```diff
@@ -45,11 +45,39 @@ hb_subset_input_create_or_fail ()
   input->unicodes = hb_set_create ();
   input->glyphs = hb_set_create ();
   input->name_ids = hb_set_create ();
+  input->drop_tables = hb_set_create ();
   input->drop_hints = false;
-  input->drop_layout = true;
   input->desubroutinize = false;
   input->retain_gids = false;
 
+  hb_tag_t default_drop_tables[] = {
+    // Layout disabled by default
+    HB_TAG ('G', 'S', 'U', 'B'),
+    HB_TAG ('G', 'P', 'O', 'S'),
+    HB_TAG ('G', 'D', 'E', 'F'),
+
+    // Copied from fontTools:
+    HB_TAG ('B', 'A', 'S', 'E'),
+    HB_TAG ('J', 'S', 'T', 'F'),
+    HB_TAG ('D', 'S', 'I', 'G'),
+    HB_TAG ('E', 'B', 'D', 'T'),
+    HB_TAG ('E', 'B', 'L', 'C'),
+    HB_TAG ('E', 'B', 'S', 'C'),
+    HB_TAG ('S', 'V', 'G', ' '),
+    HB_TAG ('P', 'C', 'L', 'T'),
+    HB_TAG ('L', 'T', 'S', 'H'),
+    // Graphite tables
+    HB_TAG ('F', 'e', 'a', 't'),
+    HB_TAG ('G', 'l', 'a', 't'),
+    HB_TAG ('G', 'l', 'o', 'c'),
+    HB_TAG ('S', 'i', 'l', 'f'),
+    HB_TAG ('S', 'i', 'l', 'l'),
+    // Colour
+    HB_TAG ('s', 'b', 'i', 'x')
```

Comment:
I think `CBDT`, `CLBC`, `COLR`, `CPAL` and `svg` should be dropped as well

Reply:
I'd like this set to mirror the default drop set found in fontTools. Do you think it would make sense to update fontTools to also drop those by default?

Reply:
I don't know the context and requirement of both but if sbix is dropped somewhere for being a color related those also should be dropped.

What worries me more is to embed a full color font into a saved PDF just because accidental use of Emoji in the page, for the reason I wished we had whitelisting instead blacklisting or dropping usually huge color font by default somehow.